src/pages/iou/ModalHeader.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {View, TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import Icon from '../../components/Icon';
+import Header from '../../components/Header';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import * as Expensicons from '../../components/Icon/Expensicons';
+import Tooltip from '../../components/Tooltip';
+import Navigation from '../../libs/Navigation/Navigation';
+
+const propTypes = {
+    /** Title of the header */
+    title: PropTypes.string.isRequired,
+
+    /** Should we show the back button? */
+    shouldShowBackButton: PropTypes.bool,
+
+    /** Callback to fire on back button press */
+    onBackButtonPress: PropTypes.func,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    shouldShowBackButton: true,
+    onBackButtonPress: () => Navigation.goBack(),
+};
+
+const ModalHeader = props => (
+    <View style={[styles.headerBar]}>
```

Comment:
There was [some inconsistency](https://github.com/Expensify/App/issues/17366) in styling Header with Back button. It was fixed by [this PR](https://github.com/Expensify/App/pull/17560/files).